### ISSUE TITLE
operator-sdk create api Error: unable to run post-scaffold tasks of "base.go.kubebuilder.io/v3" on s390x architecture

### ISSUE THREAD

**Issue by tdaleibm** [operator-framework/operator-sdk#5634]
I'm getting the below error when trying to create a go-based operator following through the example at https://sdk.operatorframework.io/docs/building-operators/golang/quickstart/ on a RHEL 8 machine with s390x architecture. 

```
operator-sdk create api --group cache --version v1alpha1 --kind Memcached --resource --controller
...
/home/tdale/memcached-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
/usr/local/go/src/net/cgo_linux.go:13:8: no such package located
Error: not all generators ran successfully
run `controller-gen object:headerFile=hack/boilerplate.go.txt paths=./... -w` to see all available markers, or `controller-gen object:headerFile=hack/boilerplate.go.txt paths=./... -h` for usage
make: *** [Makefile:95: generate] Error 1
Error: failed to create API: unable to run post-scaffold tasks of "base.go.kubebuilder.io/v3": exit status 2
```

This is the same error as mentioned in https://github.com/operator-framework/operator-sdk/issues/5284 , there the problem was said to be a versioning conflict, which cannot be the case here. I've been able to do the exact same steps with the same go and operator-sdk versions on an RHEL x86 machine.

Versions for s390x are the same versions as my working tests on x86:
```
# go version
go version go1.17.7 linux/s390x
# operator-sdk version
operator-sdk version: "v1.18.1", commit: "707240f006ecfc0bc86e5c21f6874d302992d598", kubernetes version: "1.21", go version: "go1.17.7", GOOS: "linux", GOARCH: "s390x"
```

I've tried multiple machines, and also with go version 1.16. Please let me know how I can further debug, or what needs to be done differently to create go-based s390x operators.  Thanks

**Comment by everettraven** [comment]
Hi @tdaleibm. As far as I am aware the tooling that operator-sdk wraps to generate files for a Go operator is [kubernetes-sigs/kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) which from what I can tell does not currently have a binary that targets s390x. (There is currently an open issue in kubebuilder for it here: https://github.com/kubernetes-sigs/kubebuilder/issues/2298)

Due to this I don't think you would be able to generate the structure for a s390x Go operator using operator-sdk. I am new to working with and contributing to this project, so others may have some better insight.

@camilamacedo86  do you have any insight into this issue?

**Comment by everettraven** [comment]
@tdaleibm After being informed that `operator-sdk` does indeed have a s390x release, I was able to set up an environment to test on s390x and replicate your issue on both Ubuntu and Fedora.

I was able to get it to work after making sure `gcc` was installed and running `export CGO_ENABLED=0`. I am curious, if you do the same does running `operator-sdk create api --group cache --version v1alpha1 --kind Memcached --resource --controller` end up working?
